ironfish-rust/src/transaction/tests.rs

```diff
@@ -103,6 +105,19 @@ fn test_transaction() {
         Transaction::read(&mut serialized_transaction[..].as_ref())
             .expect("should be able to deserialize valid transaction");
 
+    assert_eq!(read_back_transaction.version, TRANSACTION_VERSION);
```

Comment:
This isn't related to the PR change, I don't know if we need this added

Reply:
I can remove this, but I noticed we miss this assertion in the test.